lib/quicly.c

```diff
@@ -1097,7 +1097,9 @@ int quicly_get_stats(quicly_conn_t *conn, quicly_stats_t *stats)
     /* set or generate the non-pre-built stats fields here */
     stats->rtt = conn->egress.loss.rtt;
     stats->cc = conn->egress.cc;
-
+    if (stats->cc.cwnd_maximum < stats->cc.cwnd)
+        stats->cc.cwnd_maximum = stats->cc.cwnd;
+    stats->cc_cwnd_initial = quicly_cc_calc_initial_cwnd(conn->super.ctx->transport_params.max_udp_payload_size);
```

Comment:
How about making `cwnd_initial` part of `quicly_cc_t`, that gets set when the struct is initialized?

I am not sure if we would continue to have the guarantee that we will continue calculating initcwnd from transport parameters (consider the case when a NEW_TOKEN token carrying the CWND of the previous session is being used).